jablib/src/test/java/org/jabref/logic/quality/consistency/BibliographyConsistencyCheckTest.java

```diff
@@ -9,20 +9,30 @@
 import org.jabref.model.database.BibDatabaseContext;
 import org.jabref.model.database.BibDatabaseMode;
 import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.BibEntryTypesManager;
 import org.jabref.model.entry.field.Field;
 import org.jabref.model.entry.field.SpecialField;
 import org.jabref.model.entry.field.StandardField;
 import org.jabref.model.entry.field.UnknownField;
 import org.jabref.model.entry.field.UserSpecificCommentField;
 import org.jabref.model.entry.types.StandardEntryType;
 
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.io.TempDir;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 class BibliographyConsistencyCheckTest {
 
+    private BibEntryTypesManager entryTypesManager;
+
+    @BeforeEach
+    void setUp() {
+        // TODO: add some custom entry types for this manager and test with it
```

Comment:
Yes, please :) - The issue description at https://github.com/JabRef/jabref/issues/13794 should provide you with one.